server/routes/approvingLicences/handlers/approvalCases.ts

```diff
@@ -18,9 +20,17 @@ export default class ApprovalCaseRoutes {
     const activeCaseload = allPrisons.filter(p => p.agencyId === user.activeCaseload)
     const prisonCaseloadToDisplay = caseloadsSelected.length ? caseloadsSelected : [activeCaseload[0].agencyId]
     const searchTerm = search?.toLowerCase().trim()
-    const cases = approvalNeededView
-      ? await this.approverCaseloadService.getApprovalNeeded(user, prisonCaseloadToDisplay, searchTerm)
-      : await this.approverCaseloadService.getRecentlyApproved(user, prisonCaseloadToDisplay, searchTerm)
+    const cases = (
+      approvalNeededView
+        ? await this.approverCaseloadService.getApprovalNeeded(user, prisonCaseloadToDisplay, searchTerm)
+        : await this.approverCaseloadService.getRecentlyApproved(user, prisonCaseloadToDisplay, searchTerm)
+    ).map(c => {
+      return {
+        ...c,
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of. -->
**note:** ‏Testing in dev showed the date formatting was off - added in to format the dates as they appear in the caseload currently.